src/server.js

```diff
@@ -206,7 +211,7 @@ function reverseDnsLookup(socket, client, token) {
 			client.hostname = client.ip;
 		}
 
-		init(socket, client, token);
+		init(socket, client, remember);
```

Comment:
Is there a reason why this isn't just handled on the client? That way, we could get rid of the `remember` variable entirely and not need to pass it around at all. Since that flag comes from the client anyway, I don't see any harm in always returning the token.